src/Emogrifier.php

```diff
@@ -375,7 +375,27 @@ protected function process(\DOMDocument $xmlDocument)
             $allCss .= $this->getCssFromAllStyleNodes($xPath);
         }
 
-        $cssParts = $this->splitCssAndMediaQuery($allCss);
+        $splitCss = $this->splitCssAndMediaQuery($allCss);
+        // this is temporary pending refactoring `parseCssRules` and `copyCssWithMediaToStyleNode`
+        $cssParts = [
+            'css' => implode('', array_map(
+                function ($splitCssPart) {
+                    if ($splitCssPart['media'] === '') {
+                        return $splitCssPart['css'];
+                    }
```

Comment:
I know this is temporary, but please add `return '';`, so that the callback always returns a value (just for being explicit and have no IDE warnings ;) ).